benchmark/qwen3-asr-webgpu/results/1.7b-memory/series_report.py

```
@@ -0,0 +1,23 @@
+"""Peak and steady-state from a fleet memory series CSV (t_ms + one value column: harness_mb /
+largest_mb / used_mib). Steady state = median over the last third of the samples that lie on
+the loaded plateau (above 50 % of the peak), which excludes both the load ramp and the samples
+after the harness Chrome has exited."""
+import csv
+import statistics
+import sys
+
+with open(sys.argv[1], newline="") as f:
+    rows = [r for r in csv.DictReader(f) if r.get("t_ms")]
+if not rows:
+    print("no samples"); sys.exit(0)
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)